README.md

```diff
@@ -129,7 +129,7 @@ Once you have defined your particle you can start with the functor class.
 #### Definition
 Important parts to implement:
 * Actual force calculations: `AoSFunctor()` and all Versions of `SoAFunctor*()` 
-* Newton3 characteristics of the force: `allowsNewton3()` and `allowsNonNewton3()`
+* Newton3 characteristics of the force: `allowsNewton3()`, `allowsNonNewton3()` and `allowsMixedNewton3()`
```

Comment:
done in [4a5aece](https://github.com/AutoPas/AutoPas/pull/745/commits/4a5aece0f8222be36761514e6c229ecbf982c33f), since allowsMixedNewton3() is removed again.